src/omero_zarr/util.py

```diff
@@ -18,3 +20,16 @@ def print_status(t0: int, t: int, count: int, total: int) -> None:
         eta = "NA"
     status = f"{percent_done:.2f}% done, ETA: {eta}"
     print(status, end="\r", flush=True)
+
+
+def open_store(name: str) -> FSStore:
```

Comment:
Generally ok with this refactoring, but if it's going public we need to keep https://github.com/ome/ome-zarr-py/issues/110 and similar in mind. i.e. we may not _want_ to control this in the future.